tape/src/main/java/com/squareup/tape/QueueFile.java

```diff
@@ -586,10 +608,10 @@ public synchronized void close() throws IOException {
   static class Element {
 
     /** Length of element header in bytes. */
-    static final int HEADER_LENGTH = 4;
+    public static final int HEADER_LENGTH = 4;
```

Comment:
This shouldn't be public API

Reply:
Ya thanks. Was an old linter complaining initially.

Reply:
Done.